3ds Max/Max2Babylon/Exporter/GlobalVertex.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using Autodesk.Max;
+using Utilities;
```

Comment:
Why is this needed if no code changed in the file ?

Reply:
Not needed. I'll remove this statement